src/Uno.UI/Controls/Window.iOS.cs

```diff
@@ -293,12 +308,22 @@ public override UIView HitTest(CGPoint point, UIEvent uievent)
 			}
 		}
 
-		private void OnKeyboardWillShow(object sender, UIKeyboardEventArgs e)
+		private void OnKeyboardWillShow(object? sender, UIKeyboardEventArgs e)
 		{
+			if (e.Notification.UserInfo is null)
+			{
+				if (this.Log().IsEnabled(LogLevel.Warning))
+				{
+					this.Log().Warn("[OnKeyboardWillShow] Notification UserInfo was null");
```

Comment:
I'm not sure this will be useful information to the developer if it's not actionable. I think it would be best to be debug level instead.

Reply:
Done!